### PR TITLE
upkeep/scripts: Reorganize JS 

### PR DESCRIPTION
**Pull request by Sidsector9**
### Description of the Change

Currently the asset files don't have a structure that makes it difficult to search or develop functionality.
This PR moves the assets under `src/js/features` directory.

```
src/js/features
├── classification
│   ├── classic
│   ├── classification-button.js
│   ├── classification-toggle.js
│   ├── ibm-watson.js
│   ├── index.js
│   ├── pre-publish-classify-post.js
│   ├── previewer.js
│   ├── previewer.scss
│   └── taxonomy-controls.js
├── commands.js
├── content-resizing
│   ├── index.js
│   └── index.scss
├── excerpt-generation
│   ├── classic
│   │   ├── index.js
│   │   └── index.scss
│   ├── index.js
│   ├── maybe-excerpt-panel.js
│   └── panel.js
├── image-generation
│   ├── extend-image-block-generate-image.js
│   ├── inserter-media-category.js
│   └── media-modal
│       ├── collections
│       │   └── images.js
│       ├── index.js
│       ├── index.scss
│       ├── models
│       │   └── image.js
│       └── views
│           ├── error-message.js
│           ├── generate-image-media-upload.js
│           ├── generated-image.js
│           ├── generated-images-container.js
│           └── prompt.js
├── media-processing
│   ├── editor-ocr.js
│   └── media-upload.js
├── slot-fill
│   └── index.js
├── text-to-speech
│   ├── frontend
│   │   ├── index.js
│   │   └── index.scss
│   ├── index.js
│   └── store.js
└── title-generation
    ├── classic
    │   ├── index.js
    │   └── index.scss
    └── index.js
```

Previously, we had Classification and TTS features inside the same file, and it was done because the settings for both had to be rendered inside the Document Panel.

This PR exposes the `<ClassifaiEditorSettingPanel />`  component on the global `window` object which can be used by both internal and 3rd-party developers to extend the Panel area to add more components.

### How to test the Change

Ensure the E2E tests pass.

### Changelog Entry

> Developer - Re-organize the asset directory

### Credits

Props @Sidsector9 

### Checklist:

- [ ] I agree to follow this project's [**Code of Conduct**](https://github.com/10up/.github/blob/trunk/CODE_OF_CONDUCT.md).
- [ ] I have updated the documentation accordingly.
- [ ] I have added [Critical Flows, Test Cases, and/or End-to-End Tests](https://10up.github.io/Open-Source-Best-Practices/testing/) to cover my change.
- [ ] All new and existing tests pass.

### CODE REVIEW

File: includes/Classifai/Features/TitleGeneration.php [hunk 1]
```diff
@@ -256,13 +256,6 @@ public function enqueue_admin_assets( string $hook_suffix ) {
 					);
 				}
 			}
-
-			wp_enqueue_style(
-				'classifai-language-processing-style',
-				CLASSIFAI_PLUGIN_URL . 'dist/language-processing.css',
-				[],
-				get_asset_info( 'language-processing', 'version' ),
-			);
```
Comment (iamdharmesh) [inline includes/Classifai/Features/TitleGeneration.php:265]: It seems the language processing CSS file contains some CSS of the time generation model. Removing this minor break style of the title generation model. Need to move those CSS to `classifai-plugin-classic-title-generation.css`.

image

File: includes/Classifai/Features/ExcerptGeneration.php [hunk 2]
```diff
@@ -251,13 +251,6 @@ public function enqueue_admin_assets( string $hook_suffix ) {
 					);
 				}
 			}
-
-			wp_enqueue_style(
-				'classifai-language-processing-style',
-				CLASSIFAI_PLUGIN_URL . 'dist/language-processing.css',
-				[],
-				get_asset_info( 'language-processing', 'version' ),
-			);
```
Comment (iamdharmesh) [inline includes/Classifai/Features/ExcerptGeneration.php:260]: It seems the language processing CSS file contains some CSS of the excerpt generation model. Removing this minor break spacing above the button. 

image

**Review by iamdharmesh — CHANGES_REQUESTED** [review]
Thanks for working on this @Sidsector9, This is great. The code looks good. However, I noticed 2 minor CSS things and added notes for the same could you please help look into it? Once it is resolved we would be good to merge this.

Thank you.

**Comment by Sidsector9** [comment]
Thanks for the review @iamdharmesh, I've resolved the CSS issues in [00376be](https://github.com/10up/classifai/pull/804/commits/00376be2ab208513bc85d9616602405480e93d14)

**Review by iamdharmesh — APPROVED** [review]

**Pull request merged**